managing-os/templates/fedora-minimal.md

```diff
@@ -65,11 +65,30 @@ Use case | Description | Required steps
 **USB** | If you want USB input forwarding to use this template as the basis for a [USB](/doc/usb/) qube such as `sys-usb` | Install `qubes-input-proxy-sender`
 **VPN** | You can use this template as basis for a [VPN](/doc/vpn/) qube | Use the `dnf search "NetworkManager VPN plugin"` command to look up the VPN packages you need, based on the VPN technology you'll be using, and install them. Some GNOME related packages may be needed as well. After creation of a machine based on this template, follow the [VPN howto](/doc/vpn/#set-up-a-proxyvm-as-a-vpn-gateway-using-networkmanager) to configure it.
 **DVM Template** | If you want to use this VM as a [DVM Template](/doc/glossary/#dvm-template) | Install `perl-Encode`
-  
-  
-
+ 
 A comprehensive guide to customizing the minimal template is available [here][GUIDE]
 
+Qubes 4.0, fedora-26
+------
+In Qubes4.0 the `fedora-26-minimal` template is available and can be installed issuing the following command:
+
+~~~
+[user@dom0 ~]$ sudo qubes-dom0-update qubes-template-fedora-25-minimal
+~~~
+
+This template is slightly bigger (~600MB). All the suggestions made in the previous section apply. In addiction to this, in many situations packages from the qubes-core-agent suite will be needed to make the customized minimal template to work properly. These packages are:
+
+- `qubes-core-agent-passwordless-root`, `polkit`: By default the 'fedora-26-minimal' template doesn't have passwordless root. These two packages fix the situation.
+- `qubes-core-agent-nautilus`: This package provides integration with the Nautilus file manager (without it things like "copy to VM/open in disposable VM" will not be shown in Nautilus).
+- `qubes-core-agent-qrexec`: Qubes qrexec agent.
+- `qubes-core-agent-systemd`: Qubes unit files for SystemD init style.
+- `qubes-core-agent-sysvinit`: Qubes unit files for SysV init style or upstart.
+- `qubes-core-agent-networking`: Networking support. Useful if the template has to be used for a `sys-net` VM.
+- `qubes-core-agent-network-manager`: Integration for NetworkManager. Useful if the template has to be used for a `sys-net` VM.
+- `qubes-core-agent-dom0-updates`: Script required to handle `dom0` updates. Any template which the VM respondible for 'dom0' updates is based on must contain this package.
```

Comment:
@marmarek: Any opinion on this list of packages?

Reply:
`qubes-core-agent-qrexec`, `qubes-core-agent-systemd` are installed by default, it may be good to mark it here.
`qubes-core-agent-passwordless-root` - it may be good to point also here: https://www.qubes-os.org/doc/vm-sudo/#replacing-password-less-root-access-with-dom0-user-prompt

There is also (not strictly qubes-core-agent) `pulseaudio-qubes`, for audio support. This is also not installed by default in minimal template.

Reply:
My explanation of what these packages do is bad at least: I figured it out by trial and error for some of them, while for other I just copy-pasted what dnf search told me. Probably it is worth to have someone with a better insight to further elaborate on this.

Reply:
Besides the points I listed above, IMO your description is ok.